src/importmesh.cpp

```diff
@@ -59,20 +59,43 @@ static hEntity newPoint(EntityList *el, int id, Vector p) {
     en.actVisible = true;
     en.forceHidden = false;
 
-    en.h.v = id + en.group.v*65536;    
+    en.h.v = *id + en.group.v*65536; 
+    *id = *id+1;   
     el->Add(&en);
     return en.h;
 }
 
 // check if a vertex is unique and add it via newPoint if it is.
 static void addVertex(EntityList *el, Vector v) {
     if(el->n < 15000) {
-        int id = el->n+2;
-        newPoint(el, id, v);
+        int id = el->n;
+        newPoint(el, &id, v);
     }
 }
 
-static hEntity newLine(EntityList *el, int id, hEntity p0, hEntity p1) {
+static hEntity newNormal(EntityList *el, int *id, Quaternion normal, hEntity p) {
+    // normals have parameters, but we don't need them to make a NORMAL_N_COPY from this
+    Entity en = {};
+    en.type = Entity::Type::NORMAL_N_COPY;
+    en.extraPoints = 0;
+    en.timesApplied = 0;
+    en.group.v = 472;
+    en.actNormal = normal;
+    en.construction = false;
+    en.style.v = Style::NORMALS;
+    // to be visible we need to add a point.
+//    en.point[0] = newPoint(el, id, Vector::From(0,0,0));
```

Comment:
This is left over early code, before you added `hEntity p` as a parameter I presume.
It should be removed.

Reply:
Yes, I need to remove that.